.expeditor/scripts/build_component.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/shared.sh
+
+export HAB_AUTH_TOKEN="${ACCEPTANCE_HAB_AUTH_TOKEN}"
+export HAB_BLDR_URL="${ACCEPTANCE_HAB_BLDR_URL}"
+
+########################################################################
+
+# `component` should be the subdirectory name in `components` where a
+# particular component code resides.
+#
+# e.g. `hab` for `core/hab`, `plan-build` for `core/hab-plan-build`,
+# etc.
+component=${1}
```

Comment:
Should we throw an error if this is missing?

Reply:
Yep fixed